10/umbraco-cms/reference/configuration/installdefaultdatasettings.md

```diff
@@ -13,9 +13,9 @@ When Umbraco is installed for the first time, it creates a set of default data.
 
 In certain setups, you may want to take control over what is installed and opt-out of the creation of certain items.
 
-When working in a team and using a tool like Umbraco Deploy for sharing schema updates, when a colleague first sets up the project in their local environment, having this default data installed may not always be useful.
+When working in a team and using Umbraco Deploy for schema updates, consider the colleague's local project setup. The default data may not always be useful.
 
-For example, if a different set of languages from Umbraco's default (en-US) have been set up, it would be preferable to not have the default recreated. In other situations, certain Umbraco default Data, Member and Media Types may not be required.
+For example, if different languages are set up in Umbraco, it's better not to recreate the default language (en-US). In other situations, certain Umbraco default Data, Member and Media Types may not be required.
```

Comment:
as it mentions in the original sentence it mentions that if "a different set of languages **from** Umbraco's default (en-US) have been set up":
```suggestion
For example, if different languages are set up in Umbraco, it's better not to recreate them from the default language (en-US). In other situations, certain Umbraco default Data, Member and Media Types may not be required.
```